README.md

```diff
@@ -53,7 +54,7 @@ The following are curated lists of academic accounts on Mastodon by discipline.
 - [Historians of Science, Medicine, and Technology](https://docs.google.com/spreadsheets/d/1MKivV_PkVZBAKBXpBkxNFEvjvYvUyE-DQHtujsh6Gww/) (Google Spreadsheet; [Form to add yourself](https://docs.google.com/forms/d/e/1FAIpQLSfBvVQv5TNW6s__ufEllGXiCP53zZuKLzYOoaaHFU7p7CiqKQ/viewform))
 - [History and Philosophy of Science](https://oliviodare.github.io/mastodon-hps/)
 - [History](https://docs.google.com/spreadsheets/d/1zmuH0ffMpCNOEjsReMk5iiqGwiUvhO2TtrM8QKu98SE/htmlview#gid=844879137) (Google Spreadsheet)
-- [Human-Computer Interaction](https://directory.hci.social/)
+- [Human-Computer Interaction](https://directory.hci.social/) - See also: [HCI Directory Bot](https://hci.social/@directory_bot)
```

Comment:
Shouldn't we do this with other servers that have bots as well …? Just curious, we don't have to do that now!

Reply:
Sure, that is probably better!